### PR TITLE
Create jernej_urankar.rst

### PR DESCRIPTION
**Pull request by jerneju**

### CODE REVIEW

File: content/jernej_urankar.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Jernej Urankar
+##############
+
+:date: 2016-12-06 11:00
+:tags: sprinters
+:category: Sprinters
+:slug: jernej-urankar
+:summary: Python
+:image: /images/jernej_urankar.jpg
```
Comment (karantan) [inline content/jernej_urankar.rst:9]: You forgot to add image.

**Review by JanLikar — CHANGES_REQUESTED** [review]
Add the image

**Comment by zupo** [comment]
@jerneju: you forgot to commit the image file

**Comment by JanLikar** [comment]
Replaced by #28

**Pull request closed without merging**